packages/wallet-sdk/src/sign/SignRequestHandler.ts

```diff
@@ -55,36 +44,11 @@ export class SignRequestHandler implements RequestHandler {
     }
   }
 
-  private async eth_requestAccounts(accounts: AddressString[]): Promise<AddressString[]> {
-    if (accounts.length > 0) {
-      this.updateListener.onConnect();
```

Comment:
i believe it should be fine not to emit `connected` event for this case. 
this might be indeed correct behavior.